tdvt/samples/connector-tests/connectionbuilder.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8' ?>
```

Comment:
Not part of this PR, but we should make sure these types of TDVT tests are documented.

Reply:
The documentation of this is already  a part of the gh pages 2020.4 branch . Will open a separate PR for for getting that into gh pages branch